client_test/notebook_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Modal Labs 2022
+from pathlib import Path
+
+import pytest
+from nbclient.exceptions import CellExecutionError
+
+
+@pytest.fixture
+def notebook_runner(servicer):
+    import nbformat
+    from nbclient import NotebookClient
+
+    def runner(notebook_path: Path):
+        output_notebook_path = notebook_path.with_suffix(".output.ipynb")
```

Comment:
minor, but it would be nice to use jupytext or something similar instead of storing an .ipynb in the repo – makes it a bit easier to debug these things when things fail since you don't have to install and set up notebooks